### PR TITLE
[IMP] account: Better UX during account_accoutant_tour

### PR DESCRIPTION
**Pull request by Megaaaaaa**
Description of the issue/feature this commit addresses:

The current onboarding account tour is sometimes broken. The purpose of this commit is to fix it and make it run smoothly.

---

Desired behavior after the commit is merged :

This commit modifies the onboarding account tour to clarify the messages that are shown, add messages to certain steps and try to make it impossible for the user to be lost during its entierty.

Adding this improvement, the onboarding account tour better fills its purpose.

---

Enterprise PR : https://github.com/odoo/enterprise/pull/47306

task-3375344

---
I confirm I have signed the CLA and read the PR guidelines at www.odoo.com/submit-pr

### CODE REVIEW

File: addons/account/wizard/account_tour_upload_bill.py [hunk 1]
```diff
@@ -137,6 +150,7 @@ def apply(self):
             }
 
 
+
```
Comment (malb-odoo) [inline addons/account/wizard/account_tour_upload_bill.py:153]: useless difff :smile:

File: addons/account/wizard/account_tour_upload_bill.py [hunk 2]
```diff
@@ -147,3 +161,7 @@ def apply(self):
         purchase_journal = self.env['account.journal'].search([('type', '=', 'purchase')], limit=1)
         bill_ids = self.env['account.move'].search([('journal_id', '=', purchase_journal.id)]).ids
         return self.env['account.tour.upload.bill']._action_list_view_bill(bill_ids)
+
+class AccountTourUploadBillEmailNoAlias(models.TransientModel):
```
Comment (malb-odoo) [inline addons/account/wizard/account_tour_upload_bill.py:165]: i find it weird to add an empty class, can't we just add some logic in the xml template when there is no alias ?

File: addons/account/wizard/account_tour_upload_bill.xml [hunk 3]
```diff
@@ -39,5 +42,21 @@
...
+                    <sheet>
+                        <p>Configure your email server first and send your bill by email.</p>
+                        <a type='action' name='%(action_open_settings)d' class="btn btn-link" role="button"><i class="oi oi-fw o_button_icon oi-arrow-right"/> Configure Email Servers</a>
+                    </sheet>
+                    <footer>
+                        <button string="Close" class="btn-secondary" special="cancel" data-hotkey="x" />
+                    </footer>
+               </form>
+            </field>
+        </record>
```
Comment (malb-odoo) [inline addons/account/wizard/account_tour_upload_bill.xml:60]: With the previous comment all of that could be merge with the other template (i think)

**Review by malb-odoo — CHANGES_REQUESTED** [review]
Small comments, nice work !

**Review by malb-odoo — APPROVED** [review]
LGTM thx for the work ! (maybe the two if statement in `account_tour_upload_bill.py`could be onlined but we will lose readability so i think it's good like this)

**Comment by qdp-odoo** [comment]
needs a rebase

**Comment by qdp-odoo** [comment]
@robodoo r+

**Pull request closed without merging**